src/odemis/gui/comp/legend.py

```diff
@@ -454,7 +454,7 @@ def on_size(self, _=None):
         if self._orientation == wx.HORIZONTAL:
             self.lockBtn.SetPosition((self.ClientSize.x - 24, 0))
         else:
-            self.lockBtn.SetPosition((self.ClientSize.x - self.ClientSize.x / 2 - 12, 0))
+            self.lockBtn.SetPosition((int(self.ClientSize.x - self.ClientSize.x / 2 - 12), 0))
```

Comment:
I think you can do about the same by writing just `self.ClientSize.x // 2 - 12`.